public/javascripts/tamc-backlink.js

```diff
@@ -0,0 +1,14 @@
+/* Back link configuration */
+// store referrer value to cater for IE - https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/10474810/  */
+var docReferrer = document.referrer
+if (window.history && window.history.replaceState && typeof window.history.replaceState === 'function') {
```

Comment:
has/can somone better at javascript take a once over on this?

Reply:
I'll take a look

Reply:
Looks okay to me

Reply:
cheers @thirlz95